Protocol/ErrorMessages.xml

```diff
@@ -2680,10 +2680,10 @@
 								<Name>BelowMinimumSupportedVersion</Name>
 								<GroupDescription />
 								<Description>
-									<Format>Minimum required version '{0}' is lower than the minimum supported version '{1}'.</Format>
+									<Format>Connector minimum required DM version '{0}' is lower than the Skyline minimum supported DM version '{1}'.</Format>
```

Comment:
Not sure about the 'Connector' part. If a new name would pop up, then this would need to be renamed. As the 'NavigateTo' in DIS (or double click) goes to the MinimumRequiredVersion tag, I think it's clear with just 'Minimum required DM version'.